pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -147,14 +150,22 @@ func (c *Check) Validate() error {
 	}
 
 	// frequency must be in [1, 120] seconds
-	if c.Frequency < 1*1000 || c.Frequency > 120*1000 {
+	if c.Type() != CheckTypeTraceroute && (c.Frequency < 1*1000 || c.Frequency > 120*1000) {
 		return ErrInvalidCheckFrequency
 	}
 
 	// timeout must be in [1, 10] seconds, and it must be less than
 	// frequency (otherwise we can end up running overlapping
 	// checks)
-	if c.Timeout < 1*1000 || c.Timeout > 10*1000 || c.Timeout > c.Frequency {
+	if c.Type() == CheckTypeTraceroute && c.Timeout > 10*3000 {
+		return ErrInvalidCheckTimeout
+	}
```

Comment:
Is this validation tested at all? I'm wondering whether we could add a test for this.
I'm not too concerned though - somehow it seems we only test for `c.Timeout > c.Frequency` in the test suite so far, and have no tests for `> 10*1000` or `< 1*1000`, as far as I can tell.